src/main/java/com/doyak/reflector/payload/code/status/SuccessStatus.java

```diff
@@ -0,0 +1,32 @@
+package com.doyak.reflector.payload.code.status;
+
+import org.springframework.http.HttpStatus;
+
+import com.doyak.reflector.payload.code.BaseCode;
+import com.doyak.reflector.payload.dto.ReasonDTO;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
+public enum SuccessStatus implements BaseCode {
+
+    OK(HttpStatus.OK, "COMMON200", "성공입니다.");
```

Comment:
```suggestion
    _OK(HttpStatus.OK, "COMMON200", "성공입니다.");
```
충돌을 피하고 직관적으로 enum 상수라는 걸 구분하기 위해 앞 부분에 _ 를 추가하는 건 어떨까요?

Reply:
OK는 많이 쓰니까 구분하는게 좋겠네요..! 그런데 SUCCESS_OK 로 구분해도 괜찮을까요? 공식문서에서 Enum은 대문자+언더스코어 조합으로 구성한다고 하니까, 언더스코어로 시작하기 보다는 대문자로 시작하는게 더 좋을 것 같아서요!

Reply:
좋습니당 👍👍